src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    private static Random random = new Random();
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[getRandomArrayLength()];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+        for (Figure figure : figures) {
+            figure.draw();
+        }
+    }
+
+    public static int getRandomArrayLength() {
+        return random.nextInt(FigureSupplier.FIGURE_COUNT) + 1;
+    }
```

Comment:
```suggestion
 public static void main(String[] args) {
        FigureSupplier figureSupplier = new FigureSupplier();
        int randomArrayLength = new Random().nextInt(FigureSupplier.FIGURE_COUNT) + 1;
        Figure[] figures = new Figure[randomArrayLength];
        for (int i = 0; i < figures.length; i++) {
            figures[i] = figureSupplier.getRandomFigure();
        }
        for (Figure figure : figures) {
            figure.draw();
        }
    }
```

мне кажеться так выглядит проще